### PR TITLE
Samlin

### PR DESCRIPTION
**Pull request by u9510136**
documentation for mapper added

### CODE REVIEW

File: complex/src/Revature.Complex.Api/Controllers/AmenityController.cs [hunk 1]
```diff
@@ -15,16 +16,20 @@ namespace Revature.Complex.Api.Controllers
   public class AmenityController : ControllerBase
```
Comment (markcmoore) [inline complex/src/Revature.Complex.Api/Controllers/AmenityController.cs]: The functions in this file need documentations

File: complex/src/Revature.Complex.Api/Models/ApiRoom.cs [hunk 2]
```diff
@@ -6,11 +6,15 @@
...
+  /// (not for sending to Room service)
+  /// </summary>
   public class ApiRoom
   {
     [Required]
     public string RoomNumber { get; set; }
-    // Must have between 1 and 10 beds
     public Guid ComplexId { get; set; }
     public string Gender { get; set; }
     [Range(1, 10)]
```
Comment (markcmoore) [inline complex/src/Revature.Complex.Api/Models/ApiRoom.cs]: is this range ok being below the variables?

File: complex/src/Revature.Complex.Api/Models/ApiRoomtoSend.cs [hunk 3]
```diff
@@ -6,12 +6,15 @@
...
+  /// </summary>
   public class ApiRoomtoSend
   {
     [Required]
     public Guid RoomId { get; set; }
     public string RoomNumber { get; set; }
-    // Must have between 1 and 10 beds
     public Guid ComplexId { get; set; }
     public string Gender { get; set; }
     [Range(1, 10)]
```
Comment (markcmoore) [inline complex/src/Revature.Complex.Api/Models/ApiRoomtoSend.cs]: same here

File: complex/src/Revature.Complex.Tests/ApiTests/AmenityApiTest.cs [hunk 4]
```diff
@@ -9,6 +9,8 @@
 using Revature.Complex.Api.Controllers;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
 
 namespace Revature.Complex.Tests.ApiTests
```
Comment (markcmoore) [inline complex/src/Revature.Complex.Tests/ApiTests/AmenityApiTest.cs]: testing methods need commentary too.

File: complex/src/Revature.Complex.Api/Controllers/AmenityController.cs (hunk 1 above)
Comment (u9510136) [reply complex/src/Revature.Complex.Api/Controllers/AmenityController.cs]: All Function, test, model have documentation now.

File: complex/src/Revature.Complex.Api/Models/ApiRoom.cs (hunk 2 above)
Comment (u9510136) [reply complex/src/Revature.Complex.Api/Models/ApiRoom.cs]: it's been Removed.

File: complex/src/Revature.Complex.Api/Models/ApiRoomtoSend.cs (hunk 3 above)
Comment (u9510136) [reply complex/src/Revature.Complex.Api/Models/ApiRoomtoSend.cs]: it's been Removed

File: complex/src/Revature.Complex.Tests/ApiTests/AmenityApiTest.cs (hunk 4 above)
Comment (u9510136) [reply complex/src/Revature.Complex.Tests/ApiTests/AmenityApiTest.cs]: Added

**Pull request closed without merging**